src/test/a11y/tests/media-routes.test.ts

```diff
@@ -121,5 +121,141 @@ describe('Accessibility - Media User Routes', () => {
                 testAccessibility(url, '', true, { 'search-input': 'Invalid case number' });
             });
         });
+
+        describe('Create Media Account Page', () => {
```

Comment:
Thus is a public route not a media route

Reply:
Moved test to public route. Thanks Kian.